core/src/main/java/io/questdb/cairo/RebuildColumnBase.java

```diff
@@ -147,7 +148,6 @@ abstract protected void doReindex(
     );
 
     protected abstract boolean isSupportedColumn(RecordMetadata metadata, int columnIndex);
-
```

Comment:
This newline should be kept as it separates the methods.

Reply:
The changes have been added.